app/js/lib/actions.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+function newAddPageAction( pageName ) {
+	return {
+		type: 'ADD_PAGE',
+		payload: { name: pageName }
```

Comment:
It was like this at first but then I read about [Flux standard actions](https://github.com/acdlite/flux-standard-action) which also [exist for Reflux](https://github.com/acdlite/redux-actions).
Using the payload field was a conscious design decision.